src/pluggy/_manager.py

```diff
@@ -181,7 +192,19 @@ def parse_hookimpl_opts(self, plugin: _Plugin, name: str) -> HookimplOpts | None
         customize how hook implementation are picked up. By default, returns the
         options for items decorated with :class:`HookimplMarker`.
         """
-        method: object = getattr(plugin, name)
+
+        if _attr_is_property(plugin, name):
```

Comment:
i recently re-discovered `inspect.getattr_static`
i wonder if that one may be nice for a followup change

Reply:
When I last looked at it, the main downside of `getattr_static` was that it was very slow. Though may this has improved (I think I saw mention of that).

Reply:
marking as resolved for now to allow the PR to merge but the discussion can continue, improvements here don't need to block the rest of the PR.

The `_attr_is_property(...)` can always be swapped out for an import later if there is a better method preferred.